PersonApi/V1/Controllers/PersonApiController.cs

```diff
@@ -84,8 +85,10 @@ public async Task<IActionResult> PostNewPerson([FromBody] PersonRequestObject pe
         [HttpPatch]
         [Route("{id}")]
         [LogCall(LogLevel.Information)]
-        public async Task<IActionResult> UpdatePersonByIdAsync([FromBody] PersonRequestObject personRequestObject)
+        public async Task<IActionResult> UpdatePersonByIdAsync([FromBody] PersonRequestObject personRequestObject, [FromRoute] PersonQueryObject query)
         {
+            if (query.Id == null) return BadRequest(query.Id);
```

Comment:
This is not needed. Because you have used the `PersonQueryObject` then you get the validation that it has already for free.